### PR TITLE
Added course material acquisition instructions, and fixed some typos;…

### PR DESCRIPTION
**Pull request by AssemblyZig**
… also added a simple index.html file for a starting point.

### CODE REVIEW

**Comment by weaversa** [comment]
@AssemblyZig Will you mention that this course material can also be downloaded via the website (Click the green button and then "Download Zip")? Feel free to put that picture in.

image

File: INSTALL.md [hunk 1]
```diff
@@ -97,10 +113,10 @@ specs::Misc::Sudoku> :quit
 
 [Homebrew](https://brew.sh) is a package manager for OSX, Linux, and
 Windows Subsystem for Linux. Instructions for installing Homebrew can
-be found on Homebrew's website [https://brew.sh](https://brew.sh), and
-consist of pasting a simple command into a shell prompt.
+be found on [Homebrew's website](https://brew.sh), and consist of
+pasting a simple command into a shell prompt.
 
-Once Homebrew is installed, Cryptol (along with it's `z3` dependency)
+Once Homebrew is installed, Cryptol (along with its `z3` dependency)
```
Comment (weaversa) [inline INSTALL.md:119]: Why even have apostrophes if we're not gonna use them!

File: INSTALL.md [hunk 2]
```diff
@@ -214,24 +230,24 @@ image. We'll use
 ### Linux
 
 ```sh
-.../cryptol-course> cryptol labs/Demos/OneTimePad.md
+.../cryptol-course$ cryptol labs/Demos/OneTimePad.md
```
Comment (weaversa) [inline INSTALL.md:233]: Good catch!

File: INSTALL.md [hunk 3]
```diff
@@ -214,24 +230,24 @@ image. We'll use
...
 ...
 
 Loading module Cryptol
 Loading module labs::Demos::OneTimePad
 labs::Demos::OneTimePad> 
 ```
 
-Alternatively, you can use the `:m` or `:l` command from within 
+Alternatively, you can use the `:module` or `:load` command from within
```
Comment (weaversa) [inline INSTALL.md:241]: Well, or set the `CRYPTOLPATH` environment variable to point to the root of the repository. Might be worth mentioning.

Comment (AssemblyZig) [reply INSTALL.md]: Added CRYPTOLPATH info.

File: INSTALL.md (hunk 2 above)
Comment (AssemblyZig) [reply INSTALL.md:234]: Most recent push removed redundancy in this section.

**Pull request merged**